ios/sound.h

```diff
@@ -29,8 +29,9 @@
 #include "soundbase.h"
 #include "global.h"
 
+#import <Foundation/Foundation.h>
```

Comment:
Including Foundation.h and using BOOL made it so I could not compile the project - it causes a lot of 'Expected unqualified-id' errors during parse. 
When taking out this line and changing BOOL to bool in line 60, it works as expected for me.

Reply:
Oh that's great. Thanks a lot. So you didn't have to "Compile sources as" "Objective C++"? If so we can leave the opus lib untouched.

Is everything working on your ios device?

Reply:
Yes, now I can compile without setting "Compile sources as" "Objective C++". opus can be left untouched.
And everything still seems to be working fine on my side. Thanks a lot @jeroenvv

Reply:
You're welcome :-)